project/src/components/Layout/Header/Nav/MobileNav.tsx

```
@@ -45,7 +45,7 @@ export default function MobileNav({ currentPath, navItems }: Props) {
                     NeonSpace
                 </Link>
 
-                <button onClick={() => setIsOpen(!isOpen)} className='text-2xl' aria-label='Toggle menu'>
+                <button onClick={() => setIsOpen(!isOpen)} className='text-2xl' aria-label='Toggle menu' title='Toggle menu'>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)